### PR TITLE
MIlestone: Booklisting

### PR DESCRIPTION
**Pull request by kwambiee**
1. Added Form syntax to the homepage.
2. Use bootstrap for styling.
3. Used DOM to add title and author to display Form
4. Added books to local storage. 
5. Added add book and remove book function

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+const bookForm = document.querySelector('#form');
+// const collection = document.querySelector('.table');
```
Comment (Meltrust) [inline main.js]: - [ ] Please, kindly delete this old piece of commented code, to improve the cleanliness of the repo 💪

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@6.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.hintrc
+      - name: Webhint Report
+        run: npx hint .
```
Comment (Meltrust) [inline .github/workflows/linters.yml]: - [ ] OPTIONAL: Helpful tip:  Webhint and Lighthouse are not required for Javascript projects.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+// const displayForm = () => {
+//   document.querySelector('#button').addEventListener('click', () => {
+//     if (collection.style.display === 'none') {
+//       collection.style.display = 'block';
+//     } else {
+//       collection.style.display = 'none';
+//     }
+//   });
+// };
+// delete a book
```
Comment (Meltrust) [inline main.js]: - [ ] Please, kindly delete this old piece of commented code, to improve the cleanliness of the repo 💪

File: main.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  const books = getBooks();
+  books.splice(bk, 1);
+  save(books);
+};
+bookForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const book = new Book();
+  addBook(book);
+  bookForm.reset();
+  //   displayForm();
```
Comment (Meltrust) [inline main.js]: - [ ] Please, kindly delete this old piece of commented code, to improve the cleanliness of the repo 💪

File: style.css [hunk 5]
```diff
@@ -0,0 +1,7 @@
+/* .table {
+  display: none;
+} */
```
Comment (Meltrust) [inline style.css]: - [ ] Please, kindly delete this old piece of commented code, to improve the cleanliness of the repo 💪

File: README.md [hunk 6]
```diff
@@ -1 +1,31 @@
...
+- CSS
+- Git & Gitflow
+- Javascript
+
+## Joy Kwamboka
+
+- Github: [@kwambiee](https://github.com/kwambiee)
+- Twitter: [@kwambiee](https://twitter.com/kwambiee)
+- Linkedin: [Joy Kwamboka](https://www.linkedin.com/in/joy-kwamboka)
+- E-mail: kwambokaj2.jk@gmail.com
```
Comment (Meltrust) [inline README.md]: - [ ] Your readme is missing several sections from the required [Microverse Readme Template](https://github.com/microverseinc/readme-template).

Please, kindly implement the Microverse Readme Template to your project.  Kindly remember that all the sections from the template are required, except for "Live demo" and "Tests" if not required by the project.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @kwambiee 

_I'm sorry about your partner.  I hope he/she is back soon!_

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- I can see it took you lots of work to get this right.  Congratulations, the project works!✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**